### ISSUE TITLE
Coke Oven should only turn logs_that_burn into charcoal

### ISSUE THREAD

**Issue by RickieGamer** [BluSunrize/ImmersiveEngineering#5603]
### Explain your idea

currently the coke oven uses the #minecraft:logs tag for charcoal
I believe it should use #minecraft:logs_that_burn to exclude the nether woods

**Comment by QuickBASIC** [comment]
I strongly disagree. I feel like that even if unintentional in the original implementation (IDK if this is by design or not) it opens up possibilities for different ways to play. (Many players like living in the nether.)

I'd contend from an in-universe perspective my head canon is that the coke oven burns hotter than the vanilla furnace so can make charcoal out of otherwise not easily burned logs.

Removing this interaction homogenizes the furnace and coke oven behavior for no reason and changes a behavior that's existed for a multitude of versions.
